Corona-Warn-App/src/main/res/values-bg/strings.xml

```diff
@@ -632,8 +632,10 @@
     <string name="settings_background_priority_headline">"Автоматична актуализация на статус на риск"</string>
     <!-- YTXT: settings(background priority) - description text -->
     <string name="settings_background_priority_body">"Ако сте активирали приоритетната работа във фонов режим, приложението може да определи Вашия статус на риск по всяко време. Това ще изключи оптимизацията на потребление на батерия само за приложението Corona-Warn-App."</string>
-    <!-- XACT: settings(background priority) - illustraction description -->
-    <string name="settings_background_priority_illustration_description"/>
+    <!-- XACT: settings(background priority) - on - illustration description -->
```

Comment:
Strings should only be  added for DE and Default to avoid merge conflicts with the translation delivery.